### PR TITLE
Release preparation

### LINKED ISSUE #77: Draft release
* [ ] GitHub release.
* [x] Explain how it works in Readme.  
* [x] Say in one sentence that it requires the BrandedToken contracts.
* [x] Update changelog with link to release.
* [ ] Write and copy changelog to release.
* [x] SPDX licence format.
* [x] Update mosaic.js npm version in package.json.
* [ ] Update release date in Changelog.
* [x] Add all relevant badges.

### PR DESCRIPTION
**Pull request by abhayks1**
* Added a short release description in Changelog. 
* Updated version of `package.json`
* License is now valid SPDX expression.
* Added all relevant badges.

Fixes #77

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,6 +1,12 @@
 BrandedToken.js
 ============
 
+![Build master](https://img.shields.io/travis/OpenSTFoundation/brandedtoken.js/master.svg?label=build%20master&style=flat)
+![Build develop](https://img.shields.io/travis/OpenSTFoundation/brandedtoken.js/develop.svg?label=build%20develop&style=flat)
+![npm version](https://img.shields.io/npm/v/@openstfoundation/mosaic.js.svg?style=flat)
```
Comment (schemar) [inline README.MD]: `brandedtoken.js`, not `mosaic.js` 😉

Comment (abhayks1) [reply README.MD]: Thank you for pointing out 💯. 
Updated!

**Comment by schemar** [comment]
@benjaminbollen can you have a last check, please?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@openstfoundation/brandedtoken.js",
-  "version": "0.10.0",
+  "version": "0.10.0-beta.1",
```
Comment (sarvesh-ost) [inline package.json]:
```suggestion
  "version": "0.10.0-alpha.1",
```

**Pull request merged**